tests/unit/iotics/lib/identity/crypto/test_key_pair_helpers.py

```diff
@@ -37,6 +33,32 @@ def test_convert_seed_to_mnemonic_raises_validation_error_if_invalid_language(va
     assert 'Invalid language for mnemonic:' in str(err_wrp.value)
 
 
+def test_can_convert_mnemonic_to_seed(valid_bip39_mnemonic_english, valid_bip39_seed):
+    seed = KeyPairSecretsHelper.mnemonic_bip39_to_seed(valid_bip39_mnemonic_english, lang='english')
+    assert seed == valid_bip39_seed
+
+
+def test_can_convert_mnemonic_to_seed_with_spanish(valid_bip39_mnemonic_spanish, valid_bip39_seed):
+    seed = KeyPairSecretsHelper.mnemonic_bip39_to_seed(valid_bip39_mnemonic_spanish, lang='spanish')
+    assert seed == valid_bip39_seed
+
+
+@pytest.mark.parametrize('invalid_mnemonic,error_val', (('flee' * 10, 'Number of words must be'),
+                                                        ('flee' * 32, 'Number of words must be')))
+def test_convert_mnemonic_to_seed_raises_validation_error_if_invalid_seed(invalid_mnemonic, error_val):
+    with pytest.raises(IdentityValidationError) as err_wrp:
+        KeyPairSecretsHelper.mnemonic_bip39_to_seed(invalid_mnemonic)
+    print(err_wrp.value)
+    assert error_val in str(err_wrp.value)
+
+
+def test_convert_mnemonic_to_seed_raises_validation_error_if_invalid_language(valid_bip39_mnemonic_english):
+    with pytest.raises(IdentityValidationError) as err_wrp:
+        KeyPairSecretsHelper.mnemonic_bip39_to_seed(valid_bip39_mnemonic_english, lang='invalid_lang')
+    print(err_wrp.value)
+    assert 'Invalid language for mnemonic:' in str(err_wrp.value)
```

Comment:
Actually, we are not checking the error message in the test because of the maintenance cost. But you can check the error __cause__.
https://github.com/Iotic-Labs/iotics-identity-py/blob/f1293c35409f784bbf452057d8fba1fcf6564e62/tests/unit/iotics/lib/identity/crypto/test_proof.py#L32